File: api/v2010/outgoing_caller_id/fetch-example-1/fetch-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var outgoingCallerId = OutgoingCallerIdResource.Fetch(
            pathSid: "PNe905d7e6b410746a0fb08c57e5a186f3"
        );

        Console.WriteLine(outgoingCallerId.FriendlyName);
    }
}

File: api/v2010/outgoing_caller_id/fetch-example-1/fetch-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$outgoing_caller_id = $twilio->outgoingCallerIds("PNe905d7e6b410746a0fb08c57e5a186f3")
                             ->fetch();

print($outgoing_caller_id->friendlyName);